lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -708,26 +715,27 @@ struct FpToFpOpConversion
     if (srcElementType.isFloat8E4M3FNUZ() ||
         dstElementType.isFloat8E4M3FNUZ() ||
         (computeCapability >= 90 &&
-         (srcElementType.isFloat8E5M2() || dstElementType.isFloat8E5M2()))) {
+         ((srcElementType.isFloat8E5M2() &&
```

Comment:
I think it's starting to get annoying with all of these conditions, maybe we should make `numElements` and `inVecWidthBits`/`outVecWidthBits` part of the dictionary ?

Reply:
I agree, this is getting pretty messy